src/Groups/gsets.jl

```diff
@@ -23,6 +23,114 @@ import Hecke.orbit
 # - ...
 
 
+"""
+    acting_group(Omega::GSet)
+
+Return the group `G` acting on `Omega`.
+
+# Examples
+```jldoctest
+julia> G = symmetric_group(4);
+
+julia> acting_group(gset(G, [1])) == G
+true
+```
+"""
+function acting_group(Omega::GSet)
+  throw(ArgumentError("acting_group method for type $(typeof(Omega)) missing"))
```

Comment:
```suggestion
  throw(NotImplementedError(:acting_group, Omega))
```
and similar elsewhere